shared/tracker/index.js

```diff
@@ -131,6 +131,7 @@ export default connect(
   },
   (stateProps, dispatchProps, ownProps) => {
     return {
+      proofs: [],
```

Comment:
Proofs can be null? The type in the reducers say it isn't. Where does it become null?